lib/platform/github/index.ts

```diff
@@ -273,16 +285,16 @@ export async function initRepo({
       }
     }
     // Use default branch as PR target unless later overridden.
-    config.defaultBranch = res.body.default_branch;
+    config.defaultBranch = repo.defaultBranchRef?.name;
```

Comment:
```suggestion
    config.defaultBranch = repo.defaultBranchRef.name;
```

It's going to crash us later anyway if we have no defaultBranch, so let's remove this and cause the crash to happen immediately.